ui_framework/src/components/key_pad_menu/_key_pad_menu.scss

```diff
@@ -0,0 +1,52 @@
+// Dial menu is a phone like dial comprised of an svg icon and a title.
+
+@include component('kuiKeyPadMenu') {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: $kuiKeyPadMenuSize * 3; // Default to grid of 3
+}
+
+@include component('kuiKeyPadMenuItem') {
+  display: block;
+  padding: $kuiSize;
+  height: $kuiKeyPadMenuSize;
+  width: $kuiKeyPadMenuSize;
+  color: $kuiColorDarkShade;
+  border: $kuiBorderThin;
+  border-color: transparent;
+  border-radius: $kuiBorderRadius;
+
+  &:hover, &:focus {
```

Comment:
Hover state and focus state are different. You converged them.

![](https://d3vv6lp55qjaqc.cloudfront.net/items/0f1J2a2R3j0D0d2S3U2p/Screen%20Recording%202017-07-25%20at%2011.04%20AM.gif?X-CloudApp-Visitor-Id=59773&v=c7b51981)